src/katas.js

```diff
@@ -1,6 +1,11 @@
 // Envoyez une pull request sur ce fichier pour proposer un nouveau kata au workshop :)
 export default {
   beginner: [
+    {
```

Comment:
What do you think about making two PR ? 
- One to add the kata
- One to improve the beginners flow ?

Reply:
very good idea !

Reply:
i just want to add the katas